src/main/java/org/broadinstitute/hellbender/engine/spark/GATKSparkTool.java

```diff
@@ -119,6 +121,16 @@
             optional = true)
     protected int numReducers = 0;
 
+    @Argument(doc = "For tools that write output BAM, write an BAI file. Defaults to true.",
```

Comment:
This should use the same argument name / configuration as this:

https://github.com/broadinstitute/gatk/blob/1f6a172a34cf69841902278fa05b16669fb8cc4a/src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java#L75-L78

Reply:
Thanks for pointing this out! Fixed.